.github/scripts/sync_to_dify.py

```
@@ -0,0 +1,96 @@
+API_KEY = os.environ["DIFY_API_KEY"]
+DATASET_ID = os.environ["DIFY_DATASET_ID"]
+BASE_URL = os.environ.get("DIFY_BASE_URL", "https://api.dify.ai/v1").rstrip("/")
+
+HEADERS = {
+    "Authorization": f"Bearer {API_KEY}",
+}
+
+
+def get_changed_md_files():
+    result = subprocess.run(
+        ["git", "diff", "--name-only", "HEAD~1", "HEAD"],
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)